src/session/sessions.ts

```diff
@@ -0,0 +1,284 @@
+//#region > Imports
+//#PACKAGE "graphql"
+//## npm install "graphql"@14.6.0
+// Contains the interface for gql queries, mutations and subscriptions
+import { DocumentNode } from "graphql";
+//#PACKAGE "js-cookie"
+//## npm install "js-cookie"@2.2.1
+// A simple, lightweight JavaScript API for handling browser cookies
+import Cookies from "js-cookie";
+
+//> Session
+// Contains the base session
+import Session from "./index";
+//> Templates
+// Contains the main template for the sessions
+import { IMainTemplate } from "../templates/index";
+// Contains the SNEK template
+import SnekTemplate from "../templates/snek/index";
+//> Tasks
+// Contains SNEK tasks
+import SnekTasks from "../templates/snek/gql/tasks/index";
+//> Interfaces
+// Contains the interface for the apollo endpoint
+import { ApolloEndpoint } from "../../src/endpoints/index";
+// Contains basic session interfaces
+import { User } from "./index";
+//#endregion
+
+//#region > Classes
+/** @class A Github SubSession */
+class GithubSession extends Session {
+  /**
+   * @constructor
+   * @author Nico Schett <contact@schett.net>
+   * @param {string} sId A session name
+   * @param {Endpoint} ep A endpoint
+   * @param {IMainTemplate} template A template set
+   */
```

Comment:
Should have some sort of comment.

Initializes a Github session.

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3